### ISSUE TITLE
Multiple Errors

### ISSUE THREAD

**Issue by stefanomonteiro** [jantimon/favicons-webpack-plugin#137]
1 - EntryModuleNotFoundError: Entry module not found: Error: Can't resolve 'images/amore_paraiso-icon.png' 

2 - Field 'browser' doesn't contain a valid alias configuration

3 - my/path/node_modules doesn't exist or is not a directory

4 -  Field 'browser' doesn't contain a valid alias configuration

**Comment by cahva** [comment]
I got the same error today and I suspect the path to the icon is relative where you run the script.

1. Run webpack build using cli with icon path relative to the config - works
2. Serverside build using webpack.run() where the script is not in the same folder as webpack configs - fail

I did couple of tests and tried to pass icon path using path.resolve() but that did not work. I ran out of time and will try to find solution in the morning with fresh eyes :)
